scripts/setup.sh

```diff
@@ -7,6 +7,7 @@ VERSION_NUMBER=${VERSION_NUMBER:-"latest"}
 CURRENT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 CYPHERNODE_DIR=${CYPHERNODE_DIR:-./cyphernode}
 ORIGINAL_DIR=$(pwd)
+DOCKER_SOURCE_DIR=${DOCKER_SOURCE_DIR:-$CURRENT_DIR }
```

Comment:
All those variables are getting out of hand, any suggestion on better names?

Reply:
Doesn't look too bad from my perspective. :)

Reply:
But glad your thinking about it :+1:

Reply:
`ORIGINAL_DIR` & `CURRENT_DIR` could be better named IMO

`CURRENT_DIR` = Folder where the file is
`ORIGINAL_DIR` = I think its the current working directory of the process who called the script (not 100% sure, I need to double check)

Reply:
Any idea on better name?